lib/dialects/postgres/query-generator.js

```diff
@@ -897,6 +897,13 @@ var QueryGenerator = {
       return 'ARRAY[' + value.map(function (v) {
         return SqlString.escape(JSON.stringify(v), false, this.options.timezone, this.dialect, field);
       }, this).join(',') + ']::' + jsonType.key + '[]';
+    } else if (value && field && field.type instanceof DataTypes.GEOMETRY) {
+      var s = "ST_GeomFromGeoJSON('" + JSON.stringify(value) + "'";
+      if (field.type.srid) {
+        s += + ", " + field.type.srid;
```

Comment:
hey @mikeringrose, I think you are either missing another term in this statement, or need to remove the leading "+". I tried creating geometry with DataTypes.GEOMETRY("Point", 4326) and ended up having bad queries generated due to this line.